cibuildwheel/macos.py

```diff
@@ -21,12 +31,76 @@ def get_python_configurations(build_selector):
         PythonConfiguration(version='3.6', identifier='cp36-macosx_x86_64', url='https://www.python.org/ftp/python/3.6.8/python-3.6.8-macosx10.9.pkg'),
         PythonConfiguration(version='3.7', identifier='cp37-macosx_x86_64', url='https://www.python.org/ftp/python/3.7.6/python-3.7.6-macosx10.9.pkg'),
         PythonConfiguration(version='3.8', identifier='cp38-macosx_x86_64', url='https://www.python.org/ftp/python/3.8.1/python-3.8.1-macosx10.9.pkg'),
+        PythonConfiguration(version='2.7-v7.3.0', identifier='pp27-macosx_x86_64', url='https://bitbucket.org/pypy/pypy/downloads/pypy2.7-v7.3.0-osx64.tar.bz2'),
+        PythonConfiguration(version='3.6-v7.3.0', identifier='pp36-macosx_x86_64', url='https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.3.0-osx64.tar.bz2'),
     ]
 
     # skip builds as required
     return [c for c in python_configurations if build_selector(c.identifier)]
 
 
+def make_symlinks(dest_dir, installation_bin_path, python_executable, pip_executable):
+    assert os.path.exists(os.path.join(installation_bin_path, python_executable))
```

Comment:
Any reason why only `python` is checked to exist here? Maybe we should add `python-config` and `pip`? Or can we assume the existence of the others?

Reply:
I just moved this line from where it was before. I think the main goal of that line was to ensure that the Python installation went well. This is partially superseded by #264, I guess.

Also, I'm not sure `python-config` actually exists on PyPy. But I also don't know what it's normally used for, so I just kept it like this.

Reply:
Ah, I see. Yes, #264 will sort this out. Happy to leave this as-is then.